controls/srg_gpos/SRG-OS-000163-GPOS-00072.yml

```diff
@@ -5,10 +5,28 @@ controls:
         title: '{{{ full_name }}} must terminate all network connections associated with
     a communications session at the end of the session, or as follows: for in-band
     management sessions (privileged sessions), the session must be terminated after
-    10 minutes of inacti'
+    10 minutes of inactivity; and for user sessions (non-privileged session),
+    the session must be terminated after 15 minutes of inactivity,
+    except to fulfill documented and validated mission requirements.'
         rules:
-            - sshd_set_idle_timeout
-            - sshd_set_keepalive
-            - sshd_set_keepalive_0
+            #- sshd_set_idle_timeout
+            #- sshd_set_keepalive
+            #- sshd_set_keepalive_0
             - accounts_tmout
-        status: automated
+            - var_accounts_tmout=10_min
+        status: does not meet
```

Comment:
@Mab879 I have changed this status to "does not meet". The process says that now I need to add rationale, fix, check, mitigation text and status justification. I don't know what should be in the rationale, fix and check then. Do you have any idea? Also, I tried to add a status justification, but the problem is that in the HTML table generated by utils/create_srg_export.py the status justification text is displayed in the accounts_tmout rule table row. That looks wrong to me.

Reply:
It's probably conflicting because the status is `does not meet` and there are rules selected. We need to double check the scripts to see if they are doing the excepted thing.

But we want to be able to fulfill this requirement at some point, so whatever we put here will probably change in the final STIG, I believe you can leave without these fields blank for the time being.

Reply:
So what specifically should I remove?

Reply:
I might be wrong with regards to the code, but from my understanding when you have something `does not meet`, it implies you won't have any rules to fulfill the requirement. So selections and `does not meet` status are mutually exclusive IMO.

Reply:
Our RHEL 9 STIG process document allows that there can be rules. But yes, we can remove the remaining rules